Makefile

```diff
@@ -114,10 +113,6 @@ install-mac-brew-tools:
 		hadolint \
 		shfmt
 
-# Installs linters
```

Comment:
Wouldn't we want to keep this so people can test linting locally? but maybe put it in `./bin` local to the project and have that folder in `.gitignore`?

Reply:
I'll leave it in there.